src/GraphQL/Instrumentation/Metrics.cs

```diff
@@ -56,10 +57,10 @@ public Metrics Start(string? operationName)
         /// <summary>
         /// Sets the name of the GraphQL operation.
         /// </summary>
-        public Metrics SetOperationName(string? name)
+        public Metrics SetOperationName(ROM name)
         {
             if (Enabled && _main != null)
-                _main.Subject = name;
+                _main.Subject = name.IsEmpty ? null : (string)name;
```

Comment:
ROM never represents `null` so caller should decide how to inerpret empty strings. In that context `operationName` can never be empty string so empty ROM equals to `null` string.